### DISCUSSION TITLE
Does YOLOv5 have ability to predict on a image without any of recognised classes?

### DISCUSSION THREAD

**Discussion by Haizhuolaojisite** [ultralytics/yolov5#7412] [Q&A]
Hi All!

If I train the yolo model on an image dataset with labels as dog, cat, people, then I used this trained model to predict on a image which doesn't have any object belong to the 3 classes, like blue sky background image. 

Will the model still draw box on this image and trying to predict it has dog or cat or people? 

If so, how to avoid this situation and make the model not draw box and give a label?

Thank you for your help! I look forward to any reply.

**Comment by glenn-jocher** [comment]
No detection will be the result.
